src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -3847,4 +3847,37 @@ class Organization: ADOSVTBase
         }
         return $controlResult
     }
+    hidden [ControlResult] CheckDisableCreationOfClassicPipelines([ControlResult] $controlResult)
+    {
+       $controlResult.VerificationResult = [VerificationResult]::Failed
+       if($this.PipelineSettingsObj)
+       {
+            $orgLevelScope = $this.PipelineSettingsObj.disableClassicPipelineCreation #the name of the getter setter
+
+            if($orgLevelScope -eq $true )
+            {
+                $controlResult.AddMessage([VerificationResult]::Passed, "Creation of classic build and classic release pipelines is disabled.");
+            }
+            else{
+                $controlResult.AddMessage([VerificationResult]::Failed, "Creation of classic build and classic release pipelines is not disabled.");
+                if ($this.ControlFixBackupRequired -or $this.BaselineConfigurationRequired)
```

Comment:
Why do we need control backup here?

Reply:
Sorry, I added it by mistake. I removed it now.